app/ai/voice/agents/breeze_buddy/handlers/transport/http_handler.py

```
@@ -216,6 +219,16 @@ async def http_function_handler(
                 f"{list(config.expected_response_schema.keys())}"
             )
 
+        # In-place response transforms — applied after projection so
+        # templates can both narrow AND mutate. Channel-agnostic: voice +
+        # chat both dispatch through this handler. Only 2xx — error
+        # bodies stay raw so the LLM can read them verbatim.
+        if is_success and config.response_transforms and isinstance(data, (dict, list)):
+            try:
+                apply_response_transforms(data, config.response_transforms)
+            except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)